lib/fog/compute/openstack.rb

```diff
@@ -1,8 +1,9 @@
-
-
 module Fog
   module Compute
     class OpenStack < Fog::Service
+      SUPPORTED_VERSIONS = /v2\.0|v2\.1/
```

Comment:
Why not `freeze` here too?

Reply:
See below. We can change it either way, whatever you prefer. I don't have strong feelings on freezing.

Reply:
It's just to be consistent with other constant, I would tend to freeze the regex as well.

Reply:
sure, we'll do that